README.md

```diff
@@ -46,5 +62,14 @@ Options:
   -h, --header / --no-header  [default: no-header]
   -u, --user-facility TEXT    User facility to send data to.  [required]
   -s, --submission TEXT       Metadata submission id.  [required]
+  -?, --unique-field TEXT     The unique identifier for the submission being transformed.
```

Comment:
@sujaypatil96 is this right? what's the letter I should use here?

Reply:
```
metadata-for-user-facility-template-transformations git:(main) ✗ poetry run python etl.py --help
Usage: etl.py [OPTIONS]

  Command-line interface for creating a spreadsheet based on metadata records.

  :param submission: The ID of the metadata submission. 
  :param user_facility: The user facility to retrieve data from. 
  :param header: True if the headers should be included, False otherwise. 
  :param mapper: Path to the JSON mapper specifying column mappings.
  :param unique_field: Unique field to identify the metadata records. 
  :param output: Path to the output XLSX file.

Options:
  -s, --submission TEXT       Metadata submission id.  [required]
  -u, --user-facility TEXT    User facility to send data to.  [required]
  -h, --header / --no-header  [default: no-header]
  -m, --mapper PATH           Path to user facility specific JSON file.
                              [required]
  -uf, --unique-field TEXT    Unique field to identify the metadata records.
                              [required]
  -o, --output TEXT           Path to result output XLSX file.  [required]
  --help                      Show this message and exit.
```

I think this might be better?